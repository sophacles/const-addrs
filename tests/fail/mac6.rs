use const_addrs::mac6;

fn main() {
    let _ = mac6!("ca:fe:ca:fe:c0:ff:ee");
}
