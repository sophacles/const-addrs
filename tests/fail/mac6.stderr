error: Invalid length of 20 characters
 --> tests/fail/mac6.rs:4:19
  |
4 |     let _ = mac6!("ca:fe:ca:fe:c0:ff:ee");
  |                   ^^^^^^^^^^^^^^^^^^^^^^
